src/atoms/Indicator/Indicator.native.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/native';
+
+import View from '../View';
+import Size from '../Size/';
+import { getColor, makePxValue } from '../../_helpers/theme';
+
+const styles = {
+  backgroundColor({ variant, shape, theme }) {
+    if (shape === 'circle') {
+      return 'transparent';
+    }
+    switch (variant) {
+      case 'positive':
+        return getColor(theme, 'positive.900');
+      case 'negative':
+        return getColor(theme, 'negative.900');
+      case 'warning':
+        return getColor(theme, 'neutral.900');
+      case 'neutral':
+        return getColor(theme, 'primary.900');
+      default:
+        return getColor(theme, 'primary.900');
+    }
+  },
+  border({ variant, shape, theme }) {
+    if (shape === 'disc') {
+      return '0px';
+    }
+    switch (variant) {
+      case 'positive':
+        return `1px solid ${getColor(theme, 'positive.900')}`;
+      case 'negative':
+        return `1px solid ${getColor(theme, 'negative.900')}`;
+      case 'warning':
+        return `1px solid ${getColor(theme, 'neutral.900')}`;
+      case 'neutral':
+        return `1px solid ${getColor(theme, 'primary.900')}`;
+      default:
+        return `1px solid ${getColor(theme, 'primary.900')}`;
+    }
+  },
+};
+
+const StyledIndicator = styled(View)`
+  background-color: ${styles.backgroundColor};
+  border: ${styles.border};
+  border-radius: ${makePxValue(0.375)};
+`;
+
+const Indicator = ({ variant, shape }) => (
+  <Size height={makePxValue(0.75)} width={makePxValue(0.75)}>
```

Comment:
We need to discuss this. Paddings shouldn't be this way. It can be 2,4,8,12,16...